python/peacock/tests/input_tab/InputTreeWriter/gold/simple_diffusion_inactive.i

```diff
@@ -1,11 +1,11 @@
-inactive = 'Kernels BCs Executioner'
 [Mesh]
   type = GeneratedMesh
   dim = 2
   nx = 10
   ny = 10
 []
 
+inactive = 'Kernels BCs Executioner'
```

Comment:
Do we really want `inactive` to be somewhere in the middle? It should probably be at the top, right?

Reply:
Definitely at the top!

Reply:
That would probably be a good idea - "inactive" should probably always go first.